benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -231,12 +232,20 @@ class GraphicsBenchmarkApp
     void UpdateGUI();
     void DrawExtraInfo();
 
+    // Render to swapchain image
+    void RenderScene(grfx::SwapchainPtr swapchain, PerFrame& frame, uint32_t imageIndex);
```

Comment:
Could you add a comment of "Records commands to render * to the current frame's command buffer"

Reply:
Thanks for the suggestion, I reworded this comment (also the function has changed and been renamed)